tensorboard/webapp/metrics/views/right_pane/settings_view_component.ng.html

```diff
@@ -74,6 +74,22 @@ <h3 class="section-title">Scalars</h3>
       >Ignore outliers in chart scaling</mat-checkbox
     >
   </div>
+
+  <div class="control-row scalars-partition-x">
+    <mat-checkbox
+      [checked]="scalarPartitionX"
+      (change)="scalarPartitionXToggled.emit()"
+      >Partition non-monotonic X axis</mat-checkbox
+    >
+    <mat-icon
+      class="info"
+      svgIcon="help_outline_24px"
+      matTooltip="Non-monotonic steps can occur when reusing logdir with multiple summary
```

Comment:
> observation of bugs where it failed to reposition/disappear

IIRC, that was not the issue. Issue was that when multiple popovers are involved, there was a significant delay and performance problem. Because we are using other kinds of popver (to name a few: modal and menu), I don't think we can workaround it all the time. 

In the timeseries main content, yes, I think it is a good idea not to do without the matTooltip but I don't think we precluded it from everywhere else. In this case, with its long text, `title` simply are too illegible (at least on my DPI) so I very much favor using matTooltip (but it is not all that better; by default, it sets fontSize to 10px).

Reply:
With that said, I realized recent Chrome version has made the `title` much more salient than matTooltip so I replaced it (tho... the delay is a bit ugly).

Reply:
Thanks for switching to `title`, I definitely prefer having one type of tooltip rather than 2.  TB does have modals and menus, but imo they are justified in having a different appearance because their behavior is different from that of tooltips (they usually require clicks to open/close rather than mouse position movement).

(btw I just realized i wrote `tooltip` instead of `title`, my bad!)